index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <nav class="header">
+        <a href="#" class="logo">My Logo</a>
+        <img src="icons/menu-icon.svg" alt="menu-icon" class="menu-icon" />
+    </nav>
+    <section class="headline-wrapper">
+        <div class="headline-container">
+            <h1 class="headline-name">I’m Birhanu Glad to see you!</h1>
+            <p class="about-myself"> I’m a software developer! I can help you build a product , feature or website Look
+                through some of my work
+                and experience! If you like what you see and have a project you need coded, don’t hestiate to contact
+                me.
+            </p>
+            <div class="social-media">
+                <h2>LET’S CONNECT </h2>
+                <ul class="social-icon">
+                    <li><a href="#"> <img src="icons/twitter.svg" /> </a></li>
+                    <li><a href="#"> <img src="icons/Linkedin icon.svg" /> </a></li>
+                    <li><a href="#"> <img src="icons/m.svg" /> </a></li>
+                    <li><a href="#"> <img src="icons/github.svg" /> </a></li>
+                    <li><a href="#"> <img src="icons/peace.svg" /> </a></li>
+                </ul>
+            </div>
+        </div>
+    </section>
+</body>
+
```

Comment:
- [ ] Please use an indentation of two spaces for readability in all your documents. This is [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) and also a convention.